### ISSUE TITLE
Create single somatic MAF

### ISSUE THREAD

**Issue by evanbiederstedt** [mskcc/tempo#368]
"""
**2.	Single somatic MAF file**
•	Includes all somatic mutations (point/indels) along with annotation, filtering, FACETS CCFs, neoantigen predictions (neo_* columns from somatic_variants/neoantigen/neoantigen/*.neoantigens.maf), etc.
•	No other MAF to be retained at this time
"""

MAFs are the worst file format known to man. They are massive tab-delimited text files with dozens and dozens of columns. The first line is something like `version #2.4` https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/

RE: MAF for each TN pair
Basically, we now have 3 MAFs, there big ridiculous data.tables. For each TN pair, we should make this one MAF---add columns to the main MAF. 

This is R data.table with merge:
https://rstudio-pubs-static.s3.amazonaws.com/52230_5ae0d25125b544caab32f75f0360e775.html

RE: MAF combined across TN pairs
These MAFs can be big, so creating a massive MAF over 1000s of samples may require lots of RAM----let's think about those, or informalities
